samples/correlation-csharp/getting-started.md

```diff
@@ -0,0 +1,174 @@
+```
+
+**NOTE:** The samples targeting functions version 3. Distributed Tracing works for version 2 but not for version 1. 
+
+# Publish samples 
+
+```bash
+git clone git@github.com:Azure/azure-functions-durable-extension.git
+cd azure-functions-durable-extension
+git switch -c correlation origin/correlation
+```
+
+## Open the .sln file
+
+Go to `samples/correlation-csharp/` then open FunctionAppCorrelation.sln with Visual Studio 2019. 
+
+## Create a local.settings.json
+
+Create your `local.settings.json` for local execution. 
+You can copy and modify `local.settings.json.example.` Set the Application Insights Key. If you are not familiar with the Application Insights key, refer to [Create an Application Insights resource](https://docs.microsoft.com/en-us/azure/azure-monitor/app/create-new-resource). 
+
+```json
+{
+  "IsEncrypted": false,
+  "Values": {
+    "AzureWebJobsStorage": "UseDevelopmentStorage=true",
+    "FUNCTIONS_WORKER_RUNTIME": "dotnet",
+    "APPINSIGHTS_INSTRUMENTATIONKEY": "<YOUR_APPINSGITHS_INSTRUMENTATIONKEY_HERE>"
+  }
+}
+```
+
+## host.json
+
+Configure host.json. This JSON is the sample of the Distributed Tracing with the W3C trace context. Distributed Tracing is enabled by default. You need to configure the telemetry protocol. For more details, refer [reference](reference.md).
+
+```json
+{
+  "extensions": {
+    "durableTask": {
+      "tracing": {
+        "DistributedTracingProtocol": "W3CTraceContext"
+      }
+    }
+  },
+  "logging": {
+    "applicationInsights": {
+      "httpAutoCollectionOptions": {
+        "enableW3CDistributedTracing": true
+      } 
+    }
+  },
+  "version": "2.0"
+}
+```
+
+## Storage Emulator 
+For the local execution, you need the [Strorage Emulator](https://docs.microsoft.com/en-us/azure/storage/common/storage-use-emulator). Download it from the link and execute it before you run the functions. 
+
+## Configure NuGet 
+
+Goto the Package source on Visual Studio. Right-click FunctionAppCorrelation project, select `Manage NuGet packages...`, then click the icon. 
+
+![NuGet Manager](images/nuget-manager.png)
+
+Add Available package sources. 
+NOTE: Distributed Tracing is pre-release. We currently use myget.org until it is going to GA. 
+
+Name: azure-appservice-staging
+Source: https://www.myget.org/F/azure-appservice-staging/api/v3/index.json
+
+During pre-release, we use `Microsoft.Azure.WebJobs.Extensions.DurableTask.Telemetry` NuGet package start with Version `2.2.0-alpha`.
+
+## Push the samples to the Function App
+
+Right-click the `FunctionAppCorrelation` project, then select `Publish.`  
+
+Pick a publish target will be `Azure Functions Consumption Plan` with Select Existing with Run from package file. Click `Create Profile`
+
+![Pick a publish target](images/pick-a-publish-target.png)
+
+Then select the target function app. Then click `OK.`
+
+![App Service](images/app-service.png)
+
+## Run the samples
```

Comment:
Maintaining this duplication of this chart makes it harder to maintain. I would pick one location as the source and link to it. Or even have it as its own file and link to it in both locations.

Reply:
Done